iFixit/checkout-ui-extensions
# Add "Notes to Shipper" Attribute to Checkout

## Overview

We still use the `Notes to Shipper` attribute in our process, which apparently is one of the few pieces of data that correctly syncs from shopify to netsuite after the order is placed. This pull just replicates the functionality from `shopify-v2` where we create a blank `Notes to Shipper` attribute for checkouts which can be edited on the orders by admins later.

## QA

Make sure an order now has `Notes to Shipper` attribute like this [order](https://admin.shopify.com/store/ifixit-checkout-dev/orders/5691610333469)

Closes: #32
